app/models/video.py

```diff
@@ -1,4 +1,17 @@
 from app import db
+from app.models.rental import Rental
 
 class Video(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    __tablename__='video'
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime)
+    total_inventory = db.Column(db.Integer)
+    customers = db.relationship('Customer', secondary='rentals', backref='video')
+
+    def calculate_available_inventory(self):
```

Comment:
If you add `videos_checkout_count` to your customer model then you could do something like
```
customer.videos_checkout_count += 1
self.total_inventory -= 1
```